source/examples/time-series-forecasting-with-hpo/preprocessing_part1.ipynb

```diff
@@ -0,0 +1,3219 @@
+{
```

Comment:
<p>Given that you are calling <code>to_pandas</code> here I wonder if you could use <code>to_parquet</code> to write to GCS?  Or is the categorical column still a problem?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>Yes, you cannot save DataFrame to Parquet if it contains categorical columns.</p>